### ISSUE TITLE
[Bug]: When Hides The Navegation Menu Appears An Blank Space In Website.

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#28581]
### Steps to reproduce

1. Hide The Navegation Menu.
2. It Will Shows An Blank Space

### Expected behaviour

The Navegation Menu Hides And Not Appear The Blank Space.

### Actual behaviour

The Navegation Menu Hides And Appear The Blank Space.

### Device name

Samsung Galaxy A70

### Android version

Android 11

### Firefox release type

Firefox Nightly

### Firefox version

111.0a1

### Additional information

Website: https://aurorarf.thats.im/

Basically When Hides The Navegation Menu In This Website, It Basically Shows An Blank Space.

And Depending Of Position Of Navegation Menu, It Could Have Differents Ways Of This Bug Happens As Showed In Video Of Me Reproducing This Bug.

Link Of Video: 
https://user-images.githubusercontent.com/122238459/213259348-1cf82169-86ba-41db-ba75-1da6219ff287.mp4

**Comment by guest666c** [comment]
The Website Does Not Exist Anymore.

**Comment by cpeterso** [comment]
Thanks for reporting this issue, but without the website there is unfortunately not much we can do.
